WordGuessGame.java

```diff
@@ -31,6 +36,26 @@ public Optional<Integer> play() {
             "Your score is determined by the number of attempts remaining"
             + " after you guessed the word correctly!"
         );
-        return Optional.empty();
+		
+		String guess = EnterGuess();
+		guess = GuessData(guess);
+		NumberOfGuesses--;
+		return Optional.of(NumberOfGuesses);
     }
+	public String EnterGuess(){
+		System.out.println("enter your 5 letter guess");
+		return input.nextLine();
+}
+
+	public String GuessData(String guess){
+		int guessLength = guess.length();
+		if (guessLength != 5 || !guess.matches("[a-zA-Z0-9]{5}")){
+			System.out.println("your guess needs to be 5 letters long");
```

Comment:
If the guess is length 5 but contains non-letters, this is a misleading error message.